Mailers.md

```diff
@@ -29,6 +29,12 @@ The HTTP API has some advantages over SMTP:
 3. You don’t have to deal with MIME because Mailgun will assemble it on their side.
 4. Request libraries are available for your language of choice.
 
+### Mailgun Webhooks setup
+
+With Mailgun it is possible to setup webhooks. For example you can send at your own service
```

Comment:
Malo bolji wording bi stavio ovdje.

nes u stilu:

'If you want to track your emails more than 7 days (which is the retainer period for free mailgun) you need to setup webhooks to our mail aggregator service'